### DISCUSSION TITLE
能否公开memo，当私人微博使用呢？

### DISCUSSION THREAD

**Discussion by skyue** [usememos/memos#33] [Ideas]
如题，单条memo设置公开或不公开权限。

**Comment by boojack** [comment]
暂无，可做。

**Comment by rebron1900** [comment]
这个功能朋友圈不就有么。

**Comment by yylqing** [comment]
对于这个设置公开的，最近发现一个网站 https://metanotey.io/， 是可以添加人员一起编辑分享一个 Board。
入过感兴趣可以看看。
也可以给到作者一点灵感。

**Reply by boojack** [reply]
谢谢分享（不过好像需要邀请码
目前想做成锤子便签 + 语雀小记，可惜时间很少

**Reply by lostornot** [reply]
这个就和Google keep 很像

**Comment by boojack** [comment]
@skyue 已上线，demo: https://memos.onrender.com/u/102

**Reply by rebron1900** [reply]
帅的。

**Reply by rebron1900** [reply]
不过现在设置似乎略有麻烦？需要进入View再激活眼睛更改成Public

**Reply by boojack** [reply]
之后会优化。但是从 private 改成 public 的过程肯定是需要额外的几步操作，因为 memos 主要是给自己用的。

**Reply by skyue** [reply]
@boojack 很久没关注，没想到完成度这么高了，赞。 重新搭了一个： https://memos.skyue.com
